docs/build-logs.md

```diff
@@ -93,7 +99,45 @@ src="./images/jenkins-pipeline-logs-elastic-kibana-role.png" />
 alt="Configuration - User creation in Kibana"
 src="./images/jenkins-pipeline-logs-elastic-kibana-user.png" />
 
-## Storing Jenkins Pipeline Logs in Loki
+   
+3. Ensure your project includes OTLP exporter dependency. For Maven. add the following to your `pom.xml`:
+   ```
+     <dependency>
+       <groupID>io.opentelemetry</groupID>
+       <artifactID>opentelemetry-exporter-otlp</artifactID>
+       <version>1.30.0</version>
+     </dependency>
+   ```
```

Comment:
this dependency is managed by the plugin, not sure why you add this comment here